frontend/src/app/Support/utils.tsx

```diff
@@ -0,0 +1,20 @@
+import moment from "moment";
```

Comment:
i think momentjs is [deprecated since 2020](https://github.com/moment/moment#project-status) (now just in maintenance mode). we might want to explore a different library for the same purpose, or use standard javascript functions